nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import errno
+import argparse
+
+def parse_args(args=None):
+    Description = 'Reformat {{ cookiecutter.name }} samplesheet file and check its contents.'
+    Epilog = """Example usage: python check_samplesheet.py <FILE_IN> <FILE_OUT>"""
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument('FILE_IN', help="Input samplesheet file.")
+    parser.add_argument('FILE_OUT', help="Output file.")
+    return parser.parse_args(args)
+
+
+def make_dir(path):
+    if not len(path) == 0:
+        try:
+            os.makedirs(path)
+        except OSError as exception:
+            if exception.errno != errno.EEXIST:
+                raise
+
+
+def print_error(error,line):
+    print("ERROR: Please check samplesheet -> {}\nLine: '{}'".format(error,line.strip()))
+    sys.exit(1)
+
+
+def check_samplesheet(FileIn,FileOut):
+    ## Check header
+    HEADER = ['sample', 'fastq_1', 'fastq_2']
+    fin = open(FileIn,'r')
+    header = fin.readline().strip().split(',')
+    if header != HEADER:
+        print("ERROR: Please check samplesheet header -> {} != {}".format(','.join(header),','.join(HEADER)))
+        sys.exit(1)
+
+    sampleRunDict = {}
+    while True:
+        line = fin.readline()
+        if line:
```

Comment:
```suggestion
    while line = fin.readline():
```

Reply:
```
  File "./check_samplesheet.py", line 45
    while line = fin.readline():
               ^
SyntaxError: invalid syntax
```

Reply:
Wanted to check whether you knew something I didn't! 😅

Reply:
Sorry, shooting from the hip.. Ok, try this instead then:

```suggestion
    for line in fin:
```

Reply:
Ok. Done [here](https://github.com/nf-core/tools/pull/653/commits/a5882f952b99abfa281b7f0202907e5cc1c1e55a)

Reply:
![image](https://user-images.githubusercontent.com/465550/86802591-75e6fb80-c075-11ea-955e-9b02306ffa9d.png)

Reply:
The simple things....

Reply:
🤓